maquetado/scss/application.scss

```diff
@@ -0,0 +1,34 @@
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
+
+body {
+	font-family: 'Roboto', sans-serif;
+	margin: 0;
+	padding: 0;
+}
+
+.nav {
+
+}
+
+.nav {
+	height: 90px;
+	width: 100%;
+	position: fixed;
+	box-shadow: 0px 5px 10px $grey;
+
+	.nv-logo {
+		width: 15%;
+		float: left;
+		padding-top: 20px;
+		padding-left: 25px;
+	}
+
+	.nv-text {
+		font-size: 2em;
+		float: right;
+		padding-top: 20px;
+		padding-right: 25px;
```

Comment:
add this padding to the .nav style using something like padding: 0 25px. A good rule is to think what you are trying to separate. When you need to separate a father from their childs you should use padding in the father. If you need to add distance between to elements with the same hierarchy (siblings) then you add margin to the element.